source/Table/Table.js

```diff
@@ -447,6 +454,10 @@ export default class Table extends React.PureComponent {
       rowIndex,
     });
 
+    const onClick = event => {
+      onColumnClick && onColumnClick({columnData, dataKey, event});
```

Comment:
I don't think its necessary to pass in the `event`. We should just invoke the callback with columnData and datakey

Reply:
Indeed, I am not using `event` in my handlers, but I added it to make it the same callback parameters as `onHeaderClick`